VimeoNetworking/Sources/Constants.swift

```diff
@@ -30,4 +30,4 @@ import Foundation
 public let VimeoBaseURL = URL(string: "https://api.vimeo.com")!
 
  /// Default API version to use for requests
-internal let VimeoDefaultAPIVersionString = "3.2"
+public let VimeoDefaultAPIVersionString = "3.2"
```

Comment:
why public?

Reply:
The compiler throws an error when we try to access this constant as a default argument 
in our **AppConfiguration** initializer. 

Here's the error:
```
AppConfiguration.swift:53:54: Let 'VimeoDefaultAPIVersionString' is internal and cannot be referenced from a default argument value
```

And here's the initializer:

```Swift
public init(clientIdentifier: String,
            clientSecret: String,
            scopes: [Scope],
            keychainService: String,
            keychainAccessGroup: String? = nil,
            apiVersion: String = VimeoDefaultAPIVersionString,
            baseUrl: URL = VimeoBaseURL)
{
    self.clientIdentifier = clientIdentifier
    self.clientSecret = clientSecret
    self.scopes = scopes
    self.keychainService = keychainService
    self.keychainAccessGroup = keychainAccessGroup
    self.apiVersion = apiVersion
    self.baseUrl = baseUrl
}
```

Reply:
@jasonhawkins I'm thinking more like 

```swift 
 /// Base URL for the Vimeo API
let VimeoBaseURL = URL(string: "https://api.vimeo.com")!

 /// Default API version to use for requests
let VimeoDefaultAPIVersionString = "3.2"
```

so keeping those internal, and the init would look like:

```swift
public init(clientIdentifier: String,
                clientSecret: String,
                scopes: [Scope],
                keychainService: String,
                keychainAccessGroup: String? = nil,
                apiVersion: String? = nil,
                baseUrl: URL? = nil)
    {
        self.clientIdentifier = clientIdentifier
        self.clientSecret = clientSecret
        self.scopes = scopes
        self.keychainService = keychainService
        self.keychainAccessGroup = keychainAccessGroup
        self.apiVersion = apiVersion ?? VimeoDefaultAPIVersionString
        self.baseUrl = baseUrl ?? VimeoBaseURL
    }
```